config/config.go

```diff
@@ -413,6 +413,11 @@ func validateDashboardTab(dt *configpb.DashboardTab) error {
 		}
 	}
 
+	// Max acceptable flakiness parameter should be valid.
+	if max_acceptable_flakiness := dt.GetStatusCustomizationOptions().GetMaxAcceptableFlakiness(); max_acceptable_flakiness < 0 || max_acceptable_flakiness > 100 {
+		mErr = multierror.Append(mErr, errors.New("invalid value provided for max_acceptable_flakiness"))
```

Comment:
nit: Note what the valid value should be (e.g. "between 0.0 and 100.0, exclusive.")

Reply:
Done

Reply:
Ah sorry, I meant in the error message itself, so a user knows what to set the value to. ^^

Reply:
Also I misread the logic condition, should be 'inclusive' not 'exclusive'.